api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -346,6 +350,77 @@ union OutboundType {
   loadBalancer: "loadBalancer",
 }
 
+/** The configuration that the operators of the cluster have to authenticate to Azure. */
+model OperatorsAuthenticationProfile {
+  /** Represents the information related to Azure User-Assigned managed identities needed
+   * to perform Operators authentication based on Azure User-Assigned Managed Identities */
+  managedIdentities: ManagedIdentitiesProfile;
+}
+
+/** Represents the information related to Azure User-Assigned managed identities needed
+ * to perform Operators authentication based on Azure User-Assigned Managed Identities */
+model ManagedIdentitiesProfile {
+  /** The set of Azure User-Assigned Managed Identities leveraged for the Control Plane
+   * operators of the cluster. The set of required managed identities is dependent on the
+   * Cluster's OpenShift version. */
+  @OpenAPI.extension("x-ms-identifiers", ["operatorName"])
+  controlPlaneOperators: ControlPlaneOperatorsElement[];
+
+  /** The set of Azure User-Assigned Managed Identities leveraged for the Data Plane
+   * operators of the cluster. The set of required managed identities is dependent on the
+   * Cluster's OpenShift version. */
+  @OpenAPI.extension("x-ms-identifiers", ["operatorName"])
+  dataPlaneOperators: DataPlaneOperatorsElement[];
+
+  /** The Managed Identities Data Plane Identity URL associated with the cluster. It is the URL
+   * that will be used to communicate with the Managed Identities Resource Provider (MI RP). */
+  @visibility("create", "read")
+  dataPlaneIdentityUrl: url;
```

Comment:
Don't need this.  It's sent in headers

Reply:
+1, unless we want to expose that to the end-users

Reply:
Looks like it's `x-ms-identity-url` from step 6 of [How to onboard Managed Identities for your resource type](https://eng.ms/docs/products/arm/rbac/managed_identities/msionboardingsteps).
Please correct me if I'm wrong, the MS documentation for this is a maze.